rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PromiseWriteDataStructures.scala

```diff
@@ -122,3 +122,12 @@ case class TechniqueTemplateCopyInfo(
 ) extends HashcodeCaching {
   override def toString() = s"Promise template ${id.name}; destination ${destination}"
 }
+
+
+case class TechniqueResourceCopyInfo(
+    id         : TechniqueResourceId
+  , destination: String
+  , content    : String //template resource as a file
+) extends HashcodeCaching {
+  override def toString() = s"Promise resource ${id.name}; destination ${destination}"
```

Comment:
Can you use something else then "promise" here? Technique resource, or policy resource.

Reply:
ok, renaming previous message also then